src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1717,11 +1721,21 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
                             }
                         }
                     }
+
                     $query .= '`'.MySqlLegacySupport::getInstance()->real_escape_string($sTargetFieldName)."` = '".MySqlLegacySupport::getInstance()->real_escape_string($sqlValue)."'";
                     $setFields[$oField->name] = $sqlValue;
                 }
             }
         }
+
+        if(null !== $this->sId) {
+            $setFields = $this->filterUnchangedFieldsFromPostData(
```

Comment:
Wording: "filterFromPostData" would suggest that the actual post data is changed.

-> filterUnchangedFields ?